### DISCUSSION TITLE
Adding `*.stage.site` to staging site exceptions list

### DISCUSSION THREAD

**Discussion by boogah** [elementor/elementor#25543] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'm writing to request an update to the list of allowed staging domains on your documentation page:

https://elementor.com/help/staging-site-license-activation/

Specifically, I'd like to see `*.stage.site` subdomains added to the list. These subdomains are used for staging websites hosted by DreamHost's DreamPress managed WordPress hosting product.

As it stands, customers with single seat Elementor Pro licenses cannot use Elementor Pro on their DreamPress staging site, which is a point of frustration for them.

### Describe the solution you'd like.

Including `*.stage.site` in your licensing exception list and documentation would clarify for DreamPress users that they can utilize their Elementor Pro license for both development and staging environments.

### Describe alternatives you've considered.

As DreamPress doesn't offer customers the ability to rename their staging environment, there is no viable way to conform to your current licensing exception list.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by boogah** [comment]
Based on [other tickets](https://github.com/elementor/elementor/issues/25138) requesting additions to Elementor Pro's staging exceptions list, it seems that tagging @nicholaszein might help? 😅

**Comment by cvzero89** [comment]
This would make our lives so much easier!

**Comment by vincekay** [comment]
I would love to see this implemented, it would make editing in each environment a breeze as currently, the only workaround is transferring the licenses between two sites which is not a viable option for ease of use in my opinion.

**Comment by evandh** [comment]
Having this implemented we mean I could use DreamHost and Elementor services more effectively! Please implement this!

**Comment by EnderPar** [comment]
This would allow me to use Elementor with staging on all of my DreamHost sites... +1 from me

**Comment by Andresd8520** [comment]
Have a couple of sites on staging on DreamPress, and I'd love to see this option available. If this could be fixed, it would me amazing!

**Comment by allan0701** [comment]
This will help me use Elementor Pro on my staging sites on Dreamhost. Please have this implemented.
